syncGraph.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
+
+'''
+' @file   syncGraph.py
+' @desc   Grabs input JSON file of posts and edges, inserts into PostRelatedness ...
+'         ... the edges and inserts into PostCentrality the scores from posts.
+' @notes  The file has scrape_id's. For inserting into our DB, we need to search for our id's (seen below in queries).
+'''
+
+### Imports
+
+from datetime import datetime
+from collections import defaultdict
+import mysql.connector
+import json
+import sys
+import os
+
+
+### Constants
+
+MEWS_CONFIG_FILEPATH = 'config/inter-mews.json'
+VERBOSE = True
+
+
+### Functions
+
+def vprint(s):
+    '''
+    @desc   Prints on VERBOSE mode
+    --
+    @param  s  String to print
+    '''
+    global VERBOSE
+    if VERBOSE: print(s)
+
+
+def loadConfig(filepath):
+    '''
+    @desc   Loads the mysql config json files
+    --
+    @param  filepath  path to config file
+    '''
+    with open(filepath) as f:
+        return json.load(f)
+
+
+def connectSQL(config):
+    '''
+    @desc   Connects to mysql
+    --
+    @param  config  mysql.connector config object
+    '''
+    try:
+        return mysql.connector.connect(**config)
+    except mysql.connector.Error:
+        raise
+        return None
```

Comment:
Unreachable?

Reply:
Yeah, missed that. Will provide error message and exit. (cause it's only called once at beginning of main).